### DISCUSSION TITLE
Variable Latency Period

### DISCUSSION THREAD

**Discussion by TreyBrown3rd** [mockoon/mockoon#1035] [Ideas]
No sure if this has been proposed, but being able to vary latency within a range would be very useful (and closely simulate behavior).

**Comment by 255kb** [comment]
This is discussed here: https://github.com/mockoon/mockoon/issues/568
I would vote for the simplest implementation. if you set latency to 500 it will randomize from 0 to 500. I don't think it is worth cluttering the UI for such a specific option. 
Feel free to add your view in the issue!
